docs/dev/run_the_etl.rst

```diff
@@ -11,24 +11,182 @@ just want to use already processed data.
+
+.. code-block::
+
+    make dagit
+
+This will launch Dagit at http://localhost:3000/. You should see
+a window that looks like this:
+
+.. image:: ../images/dagit_home.png
+  :width: 800
+  :alt: Dagit home
+
+Click the hamburger button in the upper left to view the definitions,
+assets and jobs. First, you'll need to create the raw FERC databases by
+clicking on one of the ``pudl.ferc_to_sqlite`` jobs. Then select "Launchpad"
+where you can adjust the years to extract for each dataset. Then click
+"Launch Run" in the lower right hand corner of the window. Dagit will
+take you to a new window that provides information about the status of
+the job. The bottom part of the window contains dagster logs. You can
+view logs from the ``pudl`` package in the CLI window the dagit process
+is running in.
+
+Once the raw FERC databases are created by a ``pudl.ferc_to_sqlite`` job,
+you can execute the main PUDL ETL.
+
+.. note::
+
+  Make sure you've extracted the raw FERC years you are planning to process
+  with the main PUDL ETL. Jobs in the ``pudl.etl`` definition will fail if
+  the raw FERC database are missing requested years. For example, if you want
+  to process all years available in the ``pudl.etl`` definition make sure
+  you've extracted all years of the raw FERC data.
+
+Select one of the ``pudl.etl`` jobs.
+This will bring you to a window that displays all of the asset dependencies
+in the ``pudl.etl`` definition. Subsets of the ``pudl.etl`` asset graph
+are organized by asset groups. These groups are helfpul for visualizing and
+executing subsets of the asset graph.
+
+To execute the job, select ``fast_etl`` or ``full_etl`` and click "Materialize all".
+You can congifure which years to process by shift+clicking "Materialize all".
+To view the status of the run, click the date next to "Latest run:".
+
+.. image:: ../images/dagit_pudl_etl.png
+  :width: 800
+  :alt: Dagit pudl_etl
+
+You can also re-execute specific assets by selecting one or
+multiple assets in the "Overview" tab and clicking "Materialize selected".
+This is helpful if you are updating the logic of a specific asset and don't
+want to rerun the entire ETL.
+
+.. note::
+
+  Dagster will throw an ``DagsterInvalidSubsetError`` if you try to
+  re-execute a subset of assets produced by a single function. This can
+  be resolved by re-materializing the asset group of the desired asset.
+
+
+Running the ETL with CLI Commands
+---------------------------------
+You can also execute the ETL jobs using CLI commands.
+
+.. note::
+
+  We recommend using Dagit to execute the ETL as it provides additional
+  functionality for re-execution and viewing asset dependences.
+
+There are two main CLI commands for executing the PUDL processing pipeline:
+
+1. ``ferc_to_sqlite`` executes the ``pudl.ferc_to_sqlite`` dagster graph.
+   You must run this script before you can run ``pudl_etl``.
+2. ``pudl_etl`` executes the ``pudl.etl`` asset graph.
```

Comment:
Can you just run these? Do you need dagster running in the background for them to work?

Reply:
Yes you can just run these. You don't need dagit running in the background.